pkg/k8s/announcement_handlers.go

```diff
@@ -58,7 +59,11 @@ func PatchSecretHandler(kubeClient kubernetes.Interface) chan struct{} {
 				})
 
 				if _, err = kubeClient.CoreV1().Secrets(namespace).Update(context.Background(), secret, metav1.UpdateOptions{}); err != nil {
-					log.Error().Err(err).Msgf("Failed to update OwnerReference for Secret %s/%s to reference Pod %s/%s", namespace, secretName, namespace, podName)
+					// There might be conflicts when multiple controllers try to update the same resource
+					// One of the controllers will successfully update the resource, hence conflicts shoud be ignored and not treated as an error
+					if apierrors.IsConflict(err) {
```

Comment:
I assume you intended to log the error only when there's an error that's not a conflict error?
```suggestion
					if !apierrors.IsConflict(err) {
```

Reply:
I had a local change that I forgot to push